javacord-api/src/main/java/org/javacord/api/entity/AttachmentFlags.java

```diff
@@ -0,0 +1,49 @@
+package org.javacord.api.entity;
+
+public enum AttachmentFlags {
+    /**
+     * This attachment has been edited using the remix feature on mobile.
+     */
+    IS_REMIX(1 << 2),
+
+    /**
+     * The unknown message flag type.
+     */
+    UNKNOWN(-1);
+
+    private final int bitfield;
```

Comment:
This is not a bitfield, but only the value of the flag. The bitfield would be something like `010` where the `1` corresponds to a specific flag like `IS_REMIX`. And `0` would mean that another potentual flag is not applied for this attachment. So each single digit corresponds alway to one flag.
So please rename it and everywhere else in your newly added code :)
```suggestion
    private final int id;
```